utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -75,6 +89,8 @@ class HtmlParser private constructor(
         .replace("</li>", "</$CUSTOM_BULLET_LIST_TAG>")
     }
 
+    Linkify.addLinks(htmlContentTextView, Linkify.WEB_URLS)
```

Comment:
Should we be using LinkifyCompat, instead? Also, I'd expect this to be be underneath ``supportsLinks`` below since that's the only case these should be linkified.

Further, are there tests verifying that this is being used?

Reply:
Done.